src/YoutubeAtom.stories.tsx

```diff
@@ -247,7 +247,13 @@ export const GiveConsent = (): JSX.Element => {
 };
 
 export const ExternalControls = (): JSX.Element => {
-    const [videoControls, setVideoControls] = useState<VideoControls>();
+    const [videoControls, setVideoControls] = useState<VideoControls>(
+        undefined,
+    );
+
+    useEffect(() => {
```

Comment:
Just wondering why we need to reset to `undefined`?

I might have this wrong but if the state changes the `useEffect` should fire and poke the player accordingly, and we don't need to reset?

Also a minor thing if the `videoControls` represent the state rather than an 'action' I think it would make it easier to reason about the state?

Reply:
Hmm the issue here is that (at least in Storybook) if the `videoControls` state is set to `stop`, if we recommence playing the video using the youtube controls, if we press `stop` again on the external controls the state won't update and the control will have no effect. 

I'm wondering if this is a bad sign and there's a better way of exposing the controls?

Reply:
Ah I see thanks for explaining!

Two options perhaps with differing tradeoffs on correctness vs simplicity:

1) To ensure the player always triggers with a 'new' video control command we could modify `YoutubeAtom` to pass something like `${videoControl}-${Date.now()}` -> `play-1646662137207 ` to generate some uniqueness. This would ensure that each video control would update the player regardless of the preceding value. It is a little hacky but quite simple.

2) Maintain internal player state in `YoutubeAtomPlayer`. The YouTube player API provides the method: `player.getPlayerState()` which returns an enum indicating it's state. We already use this for event emitters in the `onStateChangeListener`. We could also use it to set some internal `playerState` and then the dependencies to the controls useEffect would be a combination of `videoControls` and `playerState` i.e. if those two values differ then we update the player state. This is probably more 'correct' but it would be more involved than 1).

Reply:
Yep sure makes sense 👍